python/examples/simple_send.py

```diff
@@ -37,7 +38,7 @@ def on_start(self, event):
 
     def on_sendable(self, event):
         while event.sender.credit and self.sent < self.total:
-            msg = Message(id=(self.sent+1), body={'sequence':(self.sent+1)})
+            msg = Message(id=(self.sent+1), body={'sequence': (self.sent+1)})
```

Comment:
interesting, looks like autopep8 got this line wrong, it is missing spaces around `+`

Reply:
it is only putting the spaces there with more --aggressive options added; weird, but ok